Push/LegacyPushConfiguration.swift

```diff
@@ -0,0 +1,82 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file doesn't need to be reviewed, it was renamed from `PushConfiguration` with no changes beyond the names